packages/node_modules/@webex/plugin-memberships/test/integration/spec/memberships.js

```diff
@@ -606,18 +606,35 @@ describe('plugin-memberships', function () {
       });
     });
 
-    // TODO add a test for hiding a one on one space when that
-    // feature goes GA  (see the one-on-one test in plugin-rooms)
-    /*
     describe('#hide()', () => {
-      let membership, room;
+      let membership, roomId;
 
       // We need a one-on-on space for this test
       // We create it by sending a message to the test user
-      before(() => {
+      before(() =>
         // Ensure that room is a one-on-one object
-        // Ensure that membership is the SDK users membership object
-      });
+        webex.messages.create({
+          toPersonId: user1.id,
+          text: 'This message will create a 1-1 space'
+        })
+          .then((message) => {
+            roomId = message.roomId;
+
+            // Get the test users membership
+            return webex.memberships.list({
+              roomId,
+              personId: user.id
+            });
+          })
+          .then((memberships) => {
+            // Ensure that membership is the SDK users membership object
+            membership = memberships.items[0];
+            validateMembership(membership);
+            assert((membership.personId === actor.id),
```

Comment:
These assertions are not needed since the test is for the `hide` function. These assertions should be covered in the `list` function tests. While they are good for debugging when writing the tests, we have found they cause issues when they fail because the testing framework doesn't link back the failures properly in a `before` block.

Reply:
Removed